.github/workflows/path-filter-validation.yml

```
@@ -0,0 +1,229 @@
+          echo "| Starbunk-DND | ${{ steps.filter.outputs.starbunk-dnd }} | ${{ steps.filter.outputs.starbunk-dnd == 'true' && 'Starbunk-DND only' || 'None' }} |" >> $GITHUB_STEP_SUMMARY
+          echo "| CovaBot | ${{ steps.filter.outputs.covabot }} | ${{ steps.filter.outputs.covabot == 'true' && 'CovaBot only' || 'None' }} |" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+          
+          # Calculate optimization
+          triggered_filters=0
+          if [ "${{ steps.filter.outputs.shared }}" = "true" ]; then triggered_filters=4; fi
+          if [ "${{ steps.filter.outputs.bunkbot }}" = "true" ]; then triggered_filters=$((triggered_filters + 1)); fi
+          if [ "${{ steps.filter.outputs.djcova }}" = "true" ]; then triggered_filters=$((triggered_filters + 1)); fi
+          if [ "${{ steps.filter.outputs.starbunk-dnd }}" = "true" ]; then triggered_filters=$((triggered_filters + 1)); fi
+          if [ "${{ steps.filter.outputs.covabot }}" = "true" ]; then triggered_filters=$((triggered_filters + 1)); fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)